_docs-sources/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/manual_steps.md

```diff
@@ -0,0 +1,122 @@
+---
+pagination_label: Deployment Walkthrough
+---
+
+# Manual steps
+
+Now that your infrastructure is up and running, there are some manual steps necessary before the resources meet the CIS AWS Benchmark recommendations.
+
+## 1. Revoke Gruntwork's access
+
+Now that your infrastructure is deployed, Gruntwork doesn't need access to it anymore. The access is given through the `GruntworkAccessRole` in each of the accounts. Use the `gruntwork` CLI to revoke the access, it will delete the role in each the account, [following these steps](https://github.com/gruntwork-io/gruntwork#revoking-access-to-aws):
+
+```bash
+gruntwork aws revoke \
+            --account "dev" \
+            --account "stage" \
+            --account "prod" \
+            --account "security" \
+            --account "shared" \
+            --account "logs"
+```
+
+To revoke Gruntwork's access from the "current" account—the one you are authenticated to in the terminal—use the name
+`__current__` (i.e., `--account "__current__"`).
+
+
+**Important**: Never use the `AdministratorAccess` AWS managed policy with any users, groups, or roles. It gives full access to all resources. Instead, to give access to administrators, use a policy that allows full IAM privileges (e.g. `iam:*`) on all resources. Our Landing Zone solution created a `iam-admin` group in the Security account. [More information about the existing groups in the `iam-groups` module.](https://github.com/gruntwork-io/terraform-aws-security/tree/main/modules/iam-groups)
```

Comment:
This doesn't seem related to step 1. Do we need this here at all? If so, is there some top-level stop we need to add?

Reply:
There is a missing information, because the IAM policy that `GruntworkAccessRole` uses is the `AdministratorAccess`.

>  Now that your infrastructure is deployed, Gruntwork doesn't need access to it anymore. The access is given through an IAM role called `GruntworkAccessRole` in each of the accounts through the `AdministratorAccess` policy. Use the `gruntwork` CLI to delete the IAM role and revoke access in each account by [following these steps](https://github.com/gruntwork-io/gruntwork#revoking-access-to-aws):

What do you think? I added the `through the 'AdministratorAccess' policy`.